crates/wasi-provider/src/states/starting.rs

```diff
@@ -84,30 +82,24 @@ pub(crate) type ContainerHandleMap =
 #[derive(Default, Debug)]
 /// The Kubelet is starting the Pod containers
 pub(crate) struct Starting {
-    init_handles: Arc<Mutex<ContainerHandleMap>>,
+    init_handles: ContainerHandleMap,
 }
 
 impl Starting {
-    pub(crate) fn new(init_handles: ContainerHandleMap) -> Self {
-        Starting {
-            init_handles: Arc::new(Mutex::new(init_handles)),
-        }
+    pub fn new(init_handles: ContainerHandleMap) -> Self {
```

Comment:
I'm curious on the motivation for the visibility change. Is there ever a time where someone would need to start from a Starting state? the Starting struct still has `pub(crate)` visibility which makes me think this was intentional

Reply:
It should be `pub(crate)` and I just forgot after deleting and re-adding